### DISCUSSION TITLE
评论显示问题

### DISCUSSION THREAD

**Discussion by ovolh** [HEIGE-PCloud/DoIt#225] [Q&A] [answered]
#### 更新完最新版主题，评论看不见了，设置如下：
![image](https://user-images.githubusercontent.com/33481048/133437730-0ce7420c-9df6-44a8-a9be-ab7aac7d56e5.png)
![image](https://user-images.githubusercontent.com/33481048/133437873-055ecc87-666b-486d-851f-97fc54d2d341.png)
上个版本是可以的

**Comment by HEIGE-PCloud** [comment, marked as answer]
#226
